### PR TITLE
Add deposited amount in deposit success modal

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/DepositModal/DepositModal.tsx [hunk 1]
```diff
@@ -250,6 +250,14 @@ const DepositModal: FC<DepositModalProps> = props => {
     navigate('/');
   }, [navigate]);
 
+  const depositedAmountFormatted = useMemo(() => {
+    const depositedAmount = fixedDepositStatus?.transactionData?.depositedAmount;
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/components/DepositModal/DepositModal.tsx:254]: `const { depositedAmount } = fixedDepositStatus?.transactionData?;`

**Comment by iamvukasin** [comment]
Reviewing

File: packages/tempus-client_v3/src/hooks/useFixedDeposit.ts [hunk 2]
```diff
@@ -39,7 +45,7 @@ const stream$ = fixedDeposit$.pipe(
     const { chain, poolAddress, tokenAmount, tokenTicker, tokenAddress, slippage, signer } = payload;
 
     try {
-      const contractTransaction$ = from(
+      const result$ = from(
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/hooks/useFixedDeposit.ts:48]: `fixedDeposit$` could be better

**Review by danieledamiani — APPROVED** [review]
a couple of comments

**Review by iamvukasin — APPROVED** [review]
Minor updates

File: packages/tempus-client_v3/src/components/DepositModal/DepositModal.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/DepositModal/DepositModal.tsx:254]: It's not possible to destruct possibly undefined object. (? at the end does not work).

File: packages/tempus-client_v3/src/hooks/useFixedDeposit.ts (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useFixedDeposit.ts:48]: `fixedDeposit$` is already declared in upper scope (main stream for fixed deposit).

File: packages/tempus-core-services/src/utils/getDepositedAmountFromTx.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import { Decimal, DEFAULT_DECIMAL_PRECISION } from '../datastructures';
+import ERC20ABI from '../abi/ERC20.json';
+
+export async function getDepositAmountFromTx(
+  tx: ContractTransaction,
+  depositedTokenAddress: string,
+  depositedTokenPrecision: number,
+  walletAddress: string,
+): Promise<Decimal> {
+  let depositedAmount = new Decimal(0);
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply packages/tempus-core-services/src/utils/getDepositedAmountFromTx.ts:11]: ZERO is BigNumber - I need decimal here.

Maybe we can refactor ZERO to be Decimal in next PR.

File: packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts [hunk 4]
```diff
@@ -116,6 +122,9 @@ describe('useFixedDeposit', () => {
       pending: false,
       success: true,
       contractTransaction: { hash: '0x00' },
+      transactionData: {
+        depositedAmount: new Decimal(1),
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts]: Fixed

File: packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts [hunk 5]
```diff
@@ -88,6 +91,9 @@ describe('useFixedDeposit', () => {
...
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts]: Fixed

File: packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts [hunk 6]
```diff
@@ -49,6 +49,9 @@ describe('useFixedDeposit', () => {
...
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply packages/tempus-client_v3/src/hooks/useFixedDeposit.spec.ts]: Fixed

**Pull request merged**